x-pack/plugins/observability_solution/infra/public/components/asset_details/hooks/use_process_list.ts

```diff
@@ -6,76 +6,13 @@
  */
 
 import createContainter from 'constate';
-import { fold } from 'fp-ts/lib/Either';
-import { identity } from 'fp-ts/lib/function';
-import { pipe } from 'fp-ts/lib/pipeable';
-import { useEffect } from 'react';
-import { BehaviorSubject } from 'rxjs';
-import { ProcessListAPIResponse, ProcessListAPIResponseRT } from '../../../../common/http_api';
-import { throwErrors, createPlainError } from '../../../../common/runtime_types';
-import { useHTTPRequest } from '../../../hooks/use_http_request';
 import { useMetricsDataViewContext } from '../../../containers/metrics_source';
 
 export interface SortBy {
   name: string;
   isAscending: boolean;
 }
 
-export function useProcessList(
```

Comment:
Can't you delete this entire file? Or move stuff to another one, you deleted the hook and the file is called `use_process_list`.

Reply:
We can't without further refactoring. there is this `ProcessListContextProvider` that provides data for the `processes_row_charts`